src/gie.c

```diff
@@ -1706,10 +1890,10 @@ static int pj_unitconvert_selftest (void) {
     ret = test_time(args3, 1e-6, in3, in3);   if (ret) return ret + 30;
     ret = test_time(args4, 1e-6, in4, exp4);  if (ret) return ret + 40;
     ret = test_xyz (args5, 1e-10, in5, exp5); if (ret) return ret + 50;
+    ret = test_xyz (args6, 1e-10, in6, in6);  if (ret) return ret + 50;
```

Comment:
Delete this line. It seems to be a mishap in solving a merge conflict. The same thing is tested twice here, with the first returning a unfortunate code (this is test 6, but it looks like it is test 5 if it fails). The next line is correct.

Reply:
done